extensions/cache/runtime/src/main/java/io/quarkus/cache/CacheManager.java

```diff
@@ -0,0 +1,49 @@
+package io.quarkus.cache;
+
+import java.util.Collection;
+import java.util.Optional;
+
+/**
+ * <p>
+ * Quarkus programmatic caching API entry point.
+ * </p>
+ * <p>
+ * This API can be used to access a {@link Cache} programmatically and store, retrieve or delete cache values. It shares the
+ * same caches collection than the Quarkus annotations caching API. Most {@link Cache} operations are reactive.
```

Comment:
reactive or non blocking? Avoid things like most

Reply:
I removed that sentence for now, I'm not sure it should have been there in the first place.